pages/governance/pipeline_templates.md

```diff
@@ -0,0 +1,60 @@
+# Pipeline templates
+
+> 📘 Enterprise feature
+> Pipeline templates is only available on an [Enterprise](https://buildkite.com/pricing) plan.
+
+## Overview
+
+Pipeline templates allow you to define standard pipeline step configurations to be used by all the pipelines in your organization.
+
+When a pipeline has a pipeline template assigned to it, the step configuration for that pipeline is inherited from the template and it is no longer possible to configure a custom step configuration for that pipeline via the UI.
+
+## Creating a pipeline template
+
+Only administrators can create or update pipeline templates.
+
+1. Open your Buildkite _Organization Settings_ and choose [Templates](https://buildkite.com/organizations/-/pipeline-templates)
+1. If this is your first template, click _Create a Template_. Otherwise choose _New Template_. A pipeline template is created automatically for you.
```

Comment:
Click -> Select, missing comma:

```suggestion
1. If this is your first template, select _Create a Template_. Otherwise, select _New Template_. A pipeline template is created automatically for you.
```

Reply:
@mbelton-buildkite I'm not sure "select" is the right word here, but I can see the confusion.

The challenge I had with writing this instruction is that the page you see is entirely different depending upon whether you have created a template already or not.

So when you are yet to create a template you get:
![Empty state](https://github.com/buildkite/docs/assets/673828/2f17b47a-9275-4a4f-91e8-95c7be60304b)
Where as once you have a template it looks like:
![List view](https://github.com/buildkite/docs/assets/673828/aa6f6784-9966-44a4-acc5-b51da18c2033)

Perhaps it's easiest to just describe the empty state only?

Reply:
@mbelton-buildkite oh I just saw your next comment - I guess "select" is just the phrase you use for tap/click?

In that case, ignore this 😅

Reply:
All good. I imagine most people are using Buildkite on a computer, but I think it's better to use the general term. This comes from [Microsoft's style guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click).